src/os_linux.cpp

```diff
@@ -22,6 +22,7 @@
 #include <sys/types.h>
 #include <time.h>
 #include <unistd.h>
+#include <cerrno>
```

Comment:
Include `errno.h` instead. Async-profiler traditionally prefers C headers; we may change this in the future, but then all files need to switch to C++ headers not to have a mix.

Reply:
d03b69ff641014e2a5aa4251a53b527dcb94ea0d